packages/runtime/runtime-definitions/package-lock.json

```diff
@@ -149,35 +149,47 @@
       }
     },
     "@fluidframework/container-definitions": {
-      "version": "0.45.0-49784",
-      "resolved": "https://registry.npmjs.org/@fluidframework/container-definitions/-/container-definitions-0.45.0-49784.tgz",
-      "integrity": "sha512-vjBN0UecJYUhp9MS6oSitNB6orfOf4QQKNjPuHjSVNmqPU7i3iEYNQ1fg8IN6lqfq4HhtRIovEQlFjgB36JKRg==",
+      "version": "0.47.1000",
```

Comment:
this is tricky. probably need to push a patch of container-definitions, so we can keep this on 0.45

Reply:
fyi @tylerbutler @curtisman  @jatgarg @andre4i @heliocliu

I'm not quite sure what is happening here. or why our package versions are being updated

Reply:
i feel like the expectation is that this would have been updated sooner?  it's being updated from a prerelease version and the file history looks like this file wasn't updated along with the releases as expected

Reply:
oohh. good point. this in in our lerna. we should probably just delete this package lock

Reply:
https://github.com/microsoft/FluidFramework/pull/9606/files